### ISSUE TITLE
Feature - low priority adjustable waterfall 

### ISSUE THREAD

**Issue by 49handyman** [DSheirer/sdrtrunk#698]
to help the hackrf usage and lower the display threshold for noise level.

its just aesthetics so it doesn't matter much

it works awesome even if with all my hackrf bashing. 10mhz full metro area marrs band and even working better than 3 rtl-sdr units on another laptop i run together to compare.

**Comment by DSheirer** [comment]
I agree and have wanted to add this for a while.  I'm planning a complete rewrite of the spectral display in JavaFX, so that will definitely be one of the features of the new display.
